.github/issue_template.md

```diff
@@ -0,0 +1,41 @@
+#### A note about questions vs. issues
```

Comment:
Could you use a more general opening title that would also contain further notices such as:
- **PLEASE CHECK THAT YOU ARE USING THE LATEST SCAPY VERSION INSTALLED VIA `pip install --pre scapy`**

Would be real nice

Reply:
I would suggest `pip install https://github.com/secdev/scapy `instead =)

Reply:
Is 'Things to consider'  okay? (I know - sounds really broad... like what about the FSM, yesterdays weather, minimal electro music created in the USSR in the mid 80s)

Reply:
see 9d18a11